Directory.Build.props

```diff
@@ -67,6 +67,8 @@
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
 
+    <SkipBuildingInstallers Condition="'$(SkipBuildingInstallers)' == ''">true</SkipBuildingInstallers>
```

Comment:
Do we want this in `main`, this seems nice to have as a flag but like maybe really bad as a default?